bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -127,15 +127,14 @@ static inline u64 calc_delta(u64 *prev_val, u64 *val)
 static inline u64 get_on_cpu_cycles(u32 *cpu_id)
 {
 	u64 delta, val, *prev_val;
-	struct bpf_perf_event_value c;
```

Comment:
shouldn't this have been `struct bpf_perf_event_value c = {};` to 0 out all values?

Reply:
and initialize variables `delta` etc

Reply:
I can take another pass over the bpf code and ensure everything is zero initalized. Will address that in a follow up.